### ISSUE TITLE
PIXI.BitmapText .fnt extension not supported

### ISSUE THREAD

**Issue by wojciak** [GoodBoyDigital/pixi.js#1708]
After migration from 2.x to 3.0.1 my bitmaps font stopped working. I've changed the file names from .fnt to .xml and they work fine now. Is this by design? :]

**Comment by krzysztof-o** [comment]
PIXI doesn't recognise it as an xml. If you want to use .fnt you pass xhrType  
```javascript
PIXI.loader.add('data/fonts/desyrel.fnt', {xhrType: PIXI.loaders.Resource.XHR_RESPONSE_TYPE.DOCUMENT});
```

**Comment by wojciak** [comment]
Thanks :+1:

**Comment by englercj** [comment]
So this has come up enough that I will fix it in `3.0.2` by doing custom extension support in the loader. It is being tracked in englercj/resource-loader#11
